modules/virt-attaching-vm-to-secondary-udn.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-secondary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-attaching-vm-to-secondary-udn_{context}"]                                  
+= Attaching a virtual machine to secondary user-defined networks by using the CLI
+
+You can connect a virtual machine (VM) to multiple secondary user-defined networks (UDNs) by requesting the pod network attachment, and configuring the interface binding.
```

Comment:
hm, why do you need to request the pod network attachment ? 

Quite sure you can request only secondary networks on your vm